ClinGen/clincoded
# Connects to #1144. Removal of splice site predictors table.

**Steps to test:**
1. Login and proceed to the variant selection interface of the VCI. Use 12345 as the variant_id.
2. Proceed to the "Computational" tab. Under the "Missense" sub-tab, expect to see the table content of **Splice Site Predictors**  and **Additional Information** no longer available (e.g. table content that have "IN PROGRESS" labels). Also expect to see only the 3 external links remain in the panel and they are now left-align instead of right-align.
3. Proceed to the "Silent & Intron" sub-tab under the "Computational" tab. Similarly, expect to see the table content of **Splice Site Predictors** no longer available (e.g. table content that have "IN PROGRESS" labels). Also expect to see only the 3 external links remain in the panel and they are now left-align instead of right-align.
